sdk/iot/Azure.Iot.Hub.Service/src/JobsClient.cs

```diff
@@ -4,22 +4,205 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Iot.Hub.Service.Models;
 
 namespace Azure.Iot.Hub.Service
 {
     /// <summary>
-    /// Jobs Client place holder
+    /// Jobs client to support Import/Export and Scheduled Jobs.
     /// </summary>
     public class JobsClient
     {
+        private readonly JobRestClient _jobRestClient;
+
+        protected JobsClient()
+        {
+            // This constructor only exists for mocking purposes.
+        }
+
+        internal JobsClient(JobRestClient jobRestClient)
+        {
+            Argument.AssertNotNull(jobRestClient, nameof(jobRestClient));
+
+            _jobRestClient = jobRestClient;
+        }
+
+        /// <summary>
+        /// Creates a job to export device registrations to the container.
+        /// </summary>
+        /// <param name="outputBlobContainerUri">URI containing SAS token to a blob container. This is used to output the results of the export job.</param>
```

Comment:
"This is used to output the results of the export job." is a little vague. Could this be reworded to "The exported devices will be written to this blob."?